packages/model-viewer/karma.conf.js

```diff
@@ -160,6 +160,18 @@ module.exports = function(config) {
         // domain so we go there directly instead:
         url: 'http://bs-local.com:9876'
       },
+      'iOS Safari (iOS 12)': {
```

Comment:
Weird, I thought I put this in there 😵

Reply:
I removed it and put it back in this PR; you're not crazy.